### PR TITLE
Митрохин -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by Nikita-Mitrokhin**

### CODE REVIEW

**Comment by Nikita-Mitrokhin** [comment]
@mkildishev and @SemyonBevzuk
Could you review my test, please?

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/MitrokhinNikitaTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        // Act
+        String strComplex = z.toString();
+        var expectedString = "-1.0 - 1.0i";
+
+        // Assert
+        assertEquals(expectedString, strComplex);
+    }
+
+    @Test
+    public void canCompareComplexNumbers() {
```
Comment (SemyonBevzuk) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/MitrokhinNikitaTest.java]: I think the more appropriate name is "canCompareEqualComplexNumbers".
It speaks better about the goal of the test.

Comment (Nikita-Mitrokhin) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/MitrokhinNikitaTest.java]: Ok, fixed it.

**Comment by SemyonBevzuk** [comment]
👍

**Comment by mkildishev** [comment]
👍

**Comment by Nikita-Mitrokhin** [comment]
@kirill-korniakov

**Pull request merged**